_layouts/default.html

```diff
@@ -7,173 +7,219 @@
-                <nav class="topbar-nav">
-                    <div class="topbar-link-container">
-                        <button type="button" class="topbar-close"><i class="material-icons">close</i></button>
-
-                        {% assign site_url = "/" | relative_url %}
-                        {% assign page_url = page.url | remove: '.html' %}
-
-                        {% comment %}
-                        TODO: Enumerate pages through site.pages instead of hard coded in _config.yml.
-                              Requires a bit sophisticated logic to differ between topbar and sidebar
-                              pages as well as support for page/menu hierarchies.
-                        {% for page in site.pages %}
-                        {% unless page.dir contains 'assets/' %}
-                        * {{ page.title }}
-                        {% endunless %}
-                        {% endfor %}
-                        {% endcomment %}
-
-                        <a {% if page_url == site_url %} aria-current="page" class="active" {% endif %} href="{{ site_url }}"><span>Home</span></a>
-
-                        {% for nav in site.topbar.navigation %}
-                            {% assign nav_url = nav.url | relative_url %}
-                            <a {% if page_url == nav_url %} aria-current="page" class="active" {% endif %} href="{{ nav_url }}"><span>{{ nav.title }}</span></a>
-                        {% endfor %}
-
-                        <div class="topbar-info topbar-link-right">
-                            <div class="topbar-info-contact">
-                                <a href="{{ site.github.repository_url }}/tree/master/{{ page.path }}" target="_blank" rel="noopener noreferrer" title="Edit this page on GitHub">
-                                    <svg aria-labelledby="simpleicons-github-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-                                        <title id="simpleicons-github-icon">Edit this page on GitHub</title>
-                                        <path d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12" />
-                                    </svg>
-                                </a>
-                            </div>
+
+<head>
```

Comment:
Whaaaah, please don't reindent `:all_the_things:`! 😱

Reply:
I will never think that not indenting `<head>` and `<body>` is correct. I don't know why that's Visual Studio Code's default indentation for HTML and I don't know how to disable it. I also think it's unnecessary to line-break elements that are longer than 80 characters.

Reply:
I'm guessing it's a old "convention" to reduce the overall indenting of everything by two as there will always be a `<!DOCTYPE>` followed by a `<html>` tag.